libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -580,10 +440,10 @@ protected void buildToolbar(Toolbar toolbar, ActionBar actionBar) {
             return;
         }
 
-        if (mShouldUseNewLayout) {
-            actionBar.setTitle(R.string.get_started);
-        } else {
+        if (mOptionalSiteCredsLayout) {
```

Comment:
Doesn't this change the previous behaviour a bit? If I understand correctly, previously it was just `actionBar.setTitle(R.string.get_started);` because the flag `mShouldUseNewLayout` was always true, right?

Reply:
`mShouldUseNewLayout` wasn't always `true` though. I mean, it was for WordPress, but not for Woo. When `mOptionalSiteCredsLayout` was `true`, then `mShouldUseNewLayout` would necessarily be `false`. 

That said, I believe most Woo cases would fall under the check above this one, where we check if the current login mode is `WOO_LOGIN_MODE`. I decided to leave this the way it is just in case I was missing something. Does that make sense?

Reply:
got it, thanks!